ci/deps/travis-36-locale.yaml

```diff
@@ -4,26 +4,33 @@ channels:
   - conda-forge
 dependencies:
   - beautifulsoup4
+  - blosc=1.14.3
+  - python-blosc
   - cython>=0.28.2
+  - fastparquet=0.2.1
+  - gcsfs=0.1.0
   - html5lib
   - ipython
   - jinja2
-  - lxml
-  - matplotlib
+  - lxml=3.7.0
+  - matplotlib=3.0.0
   - nomkl
   - numexpr
   - numpy
   - openpyxl
-  - psycopg2
+  - pandas-gbq=0.8.0
+  - psycopg2=2.6.2
   - pymysql=0.7.9
   - pytables
   - python-dateutil
-  - python=3.6*
+  # cannot go past python=3.6.6 for matplotlib=3.0.0 due to
+  # https://github.com/matplotlib/matplotlib/issues/12626
+  - python=3.6.6
```

Comment:
This was the reason for https://github.com/pandas-dev/pandas/pull/25554#issuecomment-476183483.

I also shifted it out of the `slow` build where failures are not immediately apparent.